libtsuba/src/RDG.cpp

```diff
@@ -211,61 +154,60 @@ tsuba::RDG::WritePartArrays(const katana::Uri& dir, tsuba::WriteGroup* desc) {
       "WritePartArrays master sz: {} mirrors sz: {} h2owned sz : {} "
       "h2owned_edges sz: {} l2u sz: {} "
       "l2g sz: {}",
-      master_nodes_.size(), mirror_nodes_.size(),
-      host_to_owned_global_node_ids_ == nullptr
+      master_nodes().size(), mirror_nodes().size(),
+      host_to_owned_global_node_ids() == nullptr
           ? 0
-          : host_to_owned_global_node_ids_->length(),
-      host_to_owned_global_edge_ids_ == nullptr
+          : host_to_owned_global_node_ids()->length(),
+      host_to_owned_global_edge_ids() == nullptr
           ? 0
-          : host_to_owned_global_edge_ids_->length(),
-      local_to_user_id_ == nullptr ? 0 : local_to_user_id_->length(),
-      local_to_global_id_ == nullptr ? 0 : local_to_global_id_->length());
+          : host_to_owned_global_edge_ids()->length(),
+      local_to_user_id() == nullptr ? 0 : local_to_user_id()->length(),
+      local_to_global_id() == nullptr ? 0 : local_to_global_id()->length());
 
-  for (size_t i = 0; i < mirror_nodes_.size(); ++i) {
-    std::string name = MirrorPropName(i);
+  for (size_t i = 0; i < mirror_nodes().size(); ++i) {
+    std::string name = RDGCore::MirrorPropName(i);
     std::string path = KATANA_CHECKED_CONTEXT(
-        StoreArrowArrayAtName(mirror_nodes_[i], dir, name, desc), "storing {}",
+        StoreArrowArrayAtName(mirror_nodes()[i], dir, name, desc), "storing {}",
         name);
     next_properties.emplace_back(tsuba::PropStorageInfo(name, path));
   }
 
-  for (size_t i = 0; i < master_nodes_.size(); ++i) {
-    std::string name = MasterPropName(i);
+  for (size_t i = 0; i < master_nodes().size(); ++i) {
+    std::string name = RDGCore::MasterPropName(i);
     std::string path = KATANA_CHECKED_CONTEXT(
-        StoreArrowArrayAtName(master_nodes_[i], dir, name, desc), "storing {}",
+        StoreArrowArrayAtName(master_nodes()[i], dir, name, desc), "storing {}",
         name);
     next_properties.emplace_back(tsuba::PropStorageInfo(name, path));
   }
 
-  if (host_to_owned_global_node_ids_ != nullptr) {
-    std::string name = kHostToOwnedGlobalNodeIDsPropName;
+  if (host_to_owned_global_node_ids() != nullptr) {
+    std::string name = RDGCore::HostToOwnedGlobalNodeIDsPropName;
```

Comment:
If it is a constant, should it start with a `k`?

Reply:
That is an interesting question. I thought that because it was in the `RDGCore` namespace it might deserve special treatment. But I suppose prefixing with 'k' is how we indicate that something is a constant so that is probably a good idea.